doc/en/installation/from_packages.rst

```diff
@@ -96,78 +93,71 @@ Perform the command:
 ::
 
    $ yum install MariaDB-server
-   $ service mysql restart
+   $ systemctl restart mysql
+
+Database management system
+--------------------------
+
+The MySQL database server should be available to complete installation (locally or not). MariaDB is recommended.
+
+It is necessary to modify **LimitNOFILE** limitation.
+Setting this option into /etc/my.cnf will NOT work.
 
+Perform instead:
+
+::
+
+   # mkdir -p  /etc/systemd/system/mariadb.service.d/
+   # echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
+   # systemctl daemon-reload
+   # service restart mysql
```

Comment:
Use standard `systemctl` as how is the whole document.

Reply:
done thanks !